lib/email/receiver.rb

```diff
@@ -370,6 +370,7 @@ def process_destination(destination, user, body, elided)
 
         create_topic(user: user,
                      raw: body,
+                     elided: elided,
```

Comment:
Good point, I've added an explicit test for when `always_show_trimmed_content` is `false`.